src/components/GenreSelect.module.css

```diff
@@ -0,0 +1,39 @@
+.nav {
+  display: flex;
+  flex-direction: row;
+  background-color: black;
+}
+
+.button {
+  color: white;
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  margin: 0 8px;
+  font-size: inherit;
+  cursor: pointer;
+  text-decoration: none;
+  position: relative;
+}
+
+.button::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: -2px;
+  width: 100%;
+  height: 1px;
+  background-color: transparent;
+}
+
+.button:hover::after {
+  background-color: transparent;
+}
+
+.button:focus {
+  outline: none;
+}
+
+.selected {
+  border-bottom: 2px solid red;
+}
```

Comment:
To be honest I have a bunch of questions regarding css. I will not describe it - the course is not about css :) Anyway, if you need it, we can disscuss))

Please create a folders (component + styles + tests) for each component inside the components folder. This will help to navigate when the components cpunt will increase.
Optionaly - create a index file for components to simplify imports section in the pages and components